File: sbillingFront/app/models/workshift.js
import DS from 'ember-data';

export default DS.Model.extend({
  date: DS.attr(),
  timestart: DS.attr(),
  timeend: DS.attr(),
  token: DS.attr(),
  actuality: DS.attr(),
  isfull: DS.attr(),

  house: DS.belongsTo('house'),
  wtype: DS.belongsTo('workshift-type'),
  responsible: DS.belongsTo('employee'),

  employeeOnWorkshift: DS.hasMany('employee-on-workshift'),
  penaltyOnShift: DS.hasMany('penalty-on-shift'),
  petition: DS.hasMany('petition'),
  roomOnWorkshift: DS.hasMany('room-on-workshift'),

  status: Ember.computed('timestart', 'timeend', function () {
    if (Ember.isEmpty(this.timeend)) {
      return 'Открыта';
    } else {
      return 'Закрыта';
    }
  }),
});

File: sBillingApi/Models/Workshift.cs
using System;
using System.Collections.Generic;

// Code scaffolded by EF Core assumes nullable reference types (NRTs) are not used or disabled.
// If you have enabled NRTs for your project, then un-comment the following line:
// #nullable disable

namespace sBillingApi.Models
{
    public partial class Workshift
    {
        public Workshift()
        {
            EmployeeOnWorkshift = new HashSet<EmployeeOnWorkshift>();
            PenaltyOnShift = new HashSet<PenaltyOnShift>();
            Petition = new HashSet<Petition>();
            RoomOnWorkshift = new HashSet<RoomOnWorkshift>();
        }

        public Guid Id { get; set; }
        public DateTime? Date { get; set; }
        public DateTime? Timestart { get; set; }
        public DateTime? Timeend { get; set; }
        public double? Token { get; set; }
        public Guid? House { get; set; }
        public Guid? Responsible { get; set; }
        public bool? Isfull { get; set; }
        public Guid? Wtype { get; set; }
        public bool? Actuality { get; set; }

        public virtual House HouseNavigation { get; set; }
        public virtual WorkshiftType WtypeNavigation { get; set; }
        public virtual ICollection<EmployeeOnWorkshift> EmployeeOnWorkshift { get; set; }
        public virtual ICollection<PenaltyOnShift> PenaltyOnShift { get; set; }
        public virtual ICollection<Petition> Petition { get; set; }
        public virtual ICollection<RoomOnWorkshift> RoomOnWorkshift { get; set; }
    }
}
